Python/Python.sublime-syntax

```diff
@@ -33,7 +33,8 @@ variables:
   identifier_continue: '[[:alnum:]_]'
   identifier: '\b[[:alpha:]_]{{identifier_continue}}*\b'
   identifier_constant: '\b(?:[\p{Lu}_][\p{Lu}_\d]*)?[\p{Lu}]{2,}[\p{Lu}_\d]*\b'  # require 2 consecutive upper-case letters
-  digitpart: (?:\d(?:_?\d)*)
+  digits: (?:\d+(?:_\d+)*)
```

Comment:
I was just writing a review and suggesting this to be `digits` when it was updated. 👀

Reply:
Hahaha.